server/graphql/v2/object/Update.js

```diff
@@ -86,6 +86,10 @@ const Update = new GraphQLObjectType({
           let membersStats = {};
           let hostedCollectivesCount = 0;
 
+          if (audience === 'NO_ONE') {
+            return null;
```

Comment:
:ice_cream: I'm not a fan of returning `null` here. Returning `null` translates as "I don't want to reply". And since they're no `id` in the response, it means no caching is possible on the frontend side. Here it's ok to reply with the same `UpdateAudienceStats` object, with `0` for every values

Reply:
@Betree Updated 👍🏻